lib/myxql/connection.ex

```diff
@@ -271,6 +271,14 @@ defmodule MyXQL.Connection do
   end
 
   ## Internals
+  defp stream_result({:error, :multiple_results}, _query, _state) do
+    raise RuntimeError,
+          "streaming stored procedures is not supported. Use MyXQL.query_many/4 and similar functions."
+  end
```

Comment:
@wojtekmach The idea here is that when streaming you can only get this type of error when using a stored procedure. The text query separated by semi-colons returns a syntax error, which I put a test for here: https://github.com/elixir-ecto/myxql/pull/152/commits/f04be981de54da404290875ceb1fecea126411da#diff-25c00486ed7c3364911a13625fb201869b4221fc96834062c8abe5f099d334daR775